lib/node_modules/@stdlib/random/base/mt19937/lib/factory.js

```diff
@@ -24,15 +61,14 @@ var max = require('@stdlib/math/base/special/max');
 var isCollection = require('@stdlib/assert/is-collection');
 var setReadOnly = require('@stdlib/utils/define-read-only-property');
 var isPositiveInteger = require('@stdlib/assert/is-positive-integer').isPrimitive;
-var INT32_MAX = require('@stdlib/constants/math/int32-max');
 var UINT32_MAX = require('@stdlib/constants/math/uint32-max');
-var randint32 = require('./rand_int32.js');
+var randuint32 = require('./rand_uint32.js');
 
 
 // VARIABLES //
 
-var NORMALIZATION_CONSTANT = (UINT32_MAX) | 0; // asm type annotation
-var MAX_SEED = (INT32_MAX - 1) | 0; // asm type annotation
+var NORMALIZATION_CONSTANT = (UINT32_MAX) >>> 0; // asm type annotation
```

Comment:
Ah. This was why the normalized `MAX` was returning the wrong number. `|0` converted `UINT32_MAX` to a signed 32-bit integer.

```javascript
(UINT32_MAX) | 0 === -1
```

As this has been fixed, you should not need to ensure `uint32` operands when setting the normalized properties.